cli/cmd/uninstall.go

```diff
@@ -196,31 +196,61 @@ func getWorkloadRolloutJsonPatch(obj client.Object, pts *v1.PodTemplateSpec) ([]
 		}
 	}
 
+	// read the original env vars (of the manifest) from the annotation
+	var origManifestEnv map[string]map[string]string
+	if obj.GetAnnotations() != nil {
+		manifestEnvAnnotation := obj.GetAnnotations()[consts.ManifestEnvOriginalValAnnotation]
+		err := json.Unmarshal([]byte(manifestEnvAnnotation), &origManifestEnv)
+		if err != nil {
+			fmt.Printf("Failed to unmarshal original env vars: %s\n", err)
```

Comment:
This message will be printed to the user console.
perhaps we can rephrase it to something more friendly to the users:

```suggestion
			fmt.Printf("Failed to rollout odigos environment variables for %s %s: %s\n", err, obj.GetName(), obj.GetKind)
```

Didn't test it, just as an example